scripts/core/editor3/reducers/tests/suggestion_style.spec.jsx

```diff
@@ -53,7 +53,7 @@ describe('editor3.reducers.suggestion.CREATE_CHANGE_STYLE_SUGGESTION', () => {
         });
     });
 
-    it('CREATE_CHANGE_STYLE_SUGGESTION - set BOLD suggestion for a bold text', () => {
+    it('should set BOLD suggestion for a bold text, the BOLD style not present anymore for selected text', () => {
```

Comment:
Sorry for picking on the details, but I think it important to have correct and properly structured sentences

* it should create a `BOLD` suggestion
* it should create a `REMOVE-BOLD` suggestion when toggling `BOLD` on text which was already bold before enabling suggestion mode
* it should not add any suggestions after toggling 'BOLD' twice
* ...

Reply:
Fix is in the next PR